generated-sample-code/samplegen_basics.js

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// DO NOT EDIT! This is a generated sample ("Request",  "samplegen_basics")
+
+// sample-metadata:
+//   title: This is the sample title
+//   description: This is the sample description
+//   usage: node samples/v1/samplegen_basics.js [--display_name "This is the default value of the display_name request field"]
+
+'use strict';
+
+function main(displayName = 'This is the default value of the display_name request field') {
+  // [START samplegen_basics]
+  /**
+  * TODO(developer): Uncomment these variables before running the sample.
+  */
+  // const displayName = 'This is the default value of the display_name request field';
+
+  // Imports the client library
+  const {ProductSearchClient} = require('@google-cloud/vision').v1;
```

Comment:
I don't think we want to advertise `.v1` for cases when it's the default version. The simple `require('@google-cloud/vision')` works. I think we need to use the exact version only if we are showing the user how to use the beta features. What do you think @JustinBeckwith @bcoe ?

Reply:
+1

I would advocate we use `synth.py` to `replace()` instances of `require('@google-cloud/<name>').<default>` with `require('@google-cloud/<name>')` in the `samples/` folder 😄

Reply:
👍 [will do](https://github.com/googleapis/nodejs-speech/commit/1342fa9406a4fce29c5fcadec9fa94fceb9b9eee).